src/main/java/seedu/address/ui/BirthdayReminderCard.java

```diff
@@ -85,6 +85,21 @@ private String generateBirthdayReminderMessage(Birthday birthday) {
         return String.format(BIRTHDAY_MESSAGE_DEFAULT, age, birthMonthDayAsString);
     }
 
+    private MonthDay accountForLeapDay(MonthDay birthMonthDay) {
+        boolean isLeapDay = birthMonthDay.equals(MonthDay.of(2, 29));
+        boolean hasPassedThisYear = birthMonthDay.isAfter(MonthDay.now());
+        int thisYear = Year.now().getValue();
+        if (!hasPassedThisYear && isLeapDay && !Year.of(thisYear).isLeap()) {
```

Comment:
I think you can simply the isLeap() part to just be Year.now().isLeap(), without having to getValue(), year it again and then check if it isLeap :)

Reply:
:)